src/Symfony/Bundle/FrameworkBundle/composer.json

```diff
@@ -45,7 +45,7 @@
         "symfony/process": "~3.4|~4.0",
         "symfony/security-core": "~3.4|~4.0",
         "symfony/security-csrf": "~3.4|~4.0",
-        "symfony/serializer": "^4.1",
+        "symfony/serializer": "^4.2",
```

Comment:
Personally, I'm ok to bump this dependency. But feature detection may be used to support both versions. WDYT @symfony/deciders?